package.json

```diff
@@ -1,20 +1,21 @@
 {
-  "name": "@optbot/template",
-  "version": "0.0.2",
-  "description": "template for starting a new package",
-  "main": "lib/start",
+  "name": "@optbot/eqpuller",
+  "version": "0.0.0",
```

Comment:
that is a version: ground 0. Since the package didn't previously exist, that's where it should start.